charts/newrelic-k8s-metrics-adapter/templates/deployment.yaml

```diff
@@ -100,6 +104,6 @@ spec:
       affinity:
         {{- . | nindent 8 }}
       {{- end }}
-      {{- if $.Values.hostNetwork }}
-      hostNetwork: {{ $.Values.hostNetwork }}
+      {{- with include "common.hostNetwork" . }}
+      hostNetwork: {{ . }}
```

Comment:
As we talked in the stand up just leaving it here as a reminder because you are switching context) we now have a [new helper that returns the value directly](https://github.com/newrelic/helm-charts/blob/cd27a2262b8b953a7e5f921de89f8c81f3ead7c4/library/common-library/templates/_hostnetwork.tpl#L31-L42)
There is a couple of examples with a [configMap](https://github.com/newrelic/helm-charts/blob/cd27a2262b8b953a7e5f921de89f8c81f3ead7c4/library/CHART-TEMPLATE/templates/example-cm-hostnetwork.yaml#L9-L12) and a [Deployment](https://github.com/newrelic/helm-charts/blob/cd27a2262b8b953a7e5f921de89f8c81f3ead7c4/library/CHART-TEMPLATE/templates/deployment.yaml#L38).

Reply:
I see it's still not merged, so I leave it to change once it is.

Reply:
Solved in last commit